index.js

```diff
@@ -259,12 +272,17 @@ Use removeArtist to do the following:
 5. Return the resulting copied array
 🌟 EXAMPLE: if removeArtist is invoked with the artists array and the number 0, it will return the resulting array with Amedeo Modigliani removed from our dataset. */
 
-function removeArtist(/*Your Code Here*/) {
-  /*Your Code Here*/
+function removeArtist(array, index) {
+  const removedArtist = [...array];
+  for (let i=0; i < removedArtist.length; i++){
```

Comment:
If you're given an index to remove within the array do you need to use the for loop?

Reply:
Now that you say it, I can see why I wouldn't need a for loop.